common/bloodlines/wc_bloodlines.txt

```diff
@@ -350,6 +350,32 @@ lireesa = { # 603 only, no related event for earlier start
 	flags = { historical_bloodline bloodline_hunter_training }
 }
 #Drust, Kvaldir and the Vrykul
+gorak_tul = { # Event-spawned, Drust warband
+	combat_rating = 8
+	monthly_character_prestige = 0.1
+	throsic_opinion = 10
+	tirassian_opinion = -10
+	bloodline_effect_necromancer_training = 1
+
+	inheritance = patrilineal
+	allow_bastards = no
+	picture = GFX_bloodlines_placeholder_symbol_2
+	flags = { historical_bloodline bloodline_necromancer_training }
+}
+ulfar = { # High Thornspeaker of the Ancient Drust
+	combat_rating = 5
+	monthly_character_piety = 0.1
+	throssian_opinion = -15
```

Comment:
Don't forget that the religion was renamed to `throsic`, so this opinion doesn't work.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/80980870-382de280-8e3a-11ea-9de0-ac050b400d0c.png)

</details>